code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -5782,3 +5782,86 @@ var/global/list/tonio_doesnt_remove=list("tonio", "blood")
 	id = THYMOL
 	description = "Thymol is used in the treatment of respiratory problems."
 	color = "#790D27" //rgb: 121, 13, 39
+
+//End of plant-specific reagents
```

Comment:
I intended that section to be for reagents that can be found in plants, not reagents obtained by mixing reagents found in plants.